otherlibs/site/src/plugins/plugins.ml

```diff
@@ -57,9 +67,23 @@ let rec get_plugin plugins requires entries =
     get_plugin plugins (value :: requires) entries
   | Rule _ :: entries -> get_plugin plugins requires entries
 
-exception Library_not_found of string
+exception Library_not_found of string list * string
+
+exception Plugin_not_found of string list * string
```

Comment:
I don't remember anymore, but do exception constructors accept record arguments? If so, I'd use that here.